app/templates/course/stage/instructions.hbs

```diff
@@ -33,7 +33,13 @@
     {{/if}}
 
     {{#if @model.courseStage.isFirst}}
-      <CoursePage::CourseStageStep::FirstStageTutorialCard @repository={{@model.activeRepository}} @courseStage={{@model.courseStage}} class="mb-6" />
+      <CoursePage::CourseStageStep::FirstStageTutorialCard
+        @repository={{@model.activeRepository}}
+        @courseStage={{@model.courseStage}}
+        @shouldHideTestRunnerCardBeforeUserHasSubmitted={{this.shouldHideTestRunnerCardBeforeUserHasSubmitted}}
```

Comment:
`FirstStageTutorialCard` doesn't even have something called `TestRunnerCard` inside it, this name `shouldHideTestRunnerCard...` feels completely wrong

Reply:
@rohitpaulk How about `shouldHideTestRunnerCardRelatedCopy`?